packages/sit-onyx/src/i18n/locales/ko-KR.json

```diff
@@ -1,6 +1,23 @@
 {
   "optional": "(선택사항)",
   "validations": {
+    "preview": {
```

Comment:
If they are missing, just omit them. Otherwise English fallback values will not work